makefile

```diff
@@ -15,21 +15,28 @@ GDC ?= gdc
 
 dmd: bin/dfmt
 
-ldc: $(SRC)
+ldc: $(SRCS)
 	$(LDC) $(LDC_FLAGS) $^ -ofbin/dfmt
 	-rm -f *.o
 
-gdc: $(SRC)
+gdc: $(SRCS)
 	$(GDC) $(GDC_FLAGS) $^ -obin/dfmt
 
 test: debug
 	cd tests && ./test.sh
 
-bin/dfmt-test: $(SRC)
+bin/dfmt-test: $(SRCS)
 	$(DC) $(DMD_TEST_FLAGS) $^ -of$@
 
-bin/dfmt: $(SRC)
+bin/dfmt: $(SRCS)
 	$(DC) $(DMD_FLAGS) $^ -of$@
 
-debug: $(SRC)
+debug: $(SRCS)
 	$(DC) $(DMD_DEBUG_FLAGS) $^ -ofbin/dfmt
+
+pkg: dmd
+	$(MAKE) -f makd/Makd.mak pkg
+
+clean:
+	@rm -rf bin/dfmt
```

Comment:
It is recommended to use `$(RM)` built in variable (which includes `-f`) and why `-r`? Is it a directory?

So you probably want to use `$(RM) bin/dfmt` instead (I wouldn't use the `@` either).